pkg/apiserver/handler.go

```diff
@@ -277,10 +278,12 @@ func handle(ginCtx *gin.Context, handler HandlerWithoutInput, input interface{},
 	resp, err := handler.Handle(reqCtx, request)
 
 	if errors.Is(err, ErrAccessForbidden) {
-		handleForbidden(ginCtx, errHandler, http.StatusForbidden, gin.Error{
-			Err:  err,
-			Type: gin.ErrorTypePrivate,
-		})
+		handleUserError(ginCtx, errHandler, http.StatusForbidden, err)
```

Comment:
what difference does this make?

Reply:
I renamed the function because it is basically `handleError`, but without attaching the error to the gin context (which would then later end up in the logger middleware) and therefore intended for errors not caused by us and therefore not needed to be resolved by us (how can we resolve a user requesting the wrong record / having no access? you resolve this in a dashboard you are building, which should be handling the error itself).
Pushed another commit as `handleError` could be simplified a lot after thinking about this